### PR TITLE
add integration kraken

### PR DESCRIPTION
**Pull request by eifinger**
## Proposed change

Add documentation for new integration kraken

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/31114
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2572
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/kraken.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Kraken
+description: Instructions on how to integrate Kraken.com sensors into Home Assistant.
+logo: kraken.svg
+ha_category:
+  - Finance
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 0.106.0
```
Comment (klaasnicolaas) [inline source/_integrations/kraken.markdown]:
```suggestion
ha_release: 0.106
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍

File: source/_integrations/kraken.markdown [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+For a list of tradable asset pairs check this [this kraken support article](https://support.kraken.com/hc/en-us/articles/201893658-Currency-pairs-available-for-trading-on-Kraken).
+
+### Extra configuration of the integration
+
+You can configure the update rate and tracked asset pairs options through the integration options flow by clicking the gear icon on the top right of the integration details page.
+
+| Option | Description |
+| -------| ----------- |
+| Update interval | Seconds between updates. |
+| Tracked Asset Pairs | Select the assets you want to track. This list is automatically updated with a list of tradable assets on kraken.com |
```
Comment (frenck) [inline source/_integrations/kraken.markdown]: Please use `{% configuration_basic %}` for this (examples can be found in the existing documentation).

File: source/_integrations/kraken.markdown [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ha_platforms:
+  - sensor
+---
+
+The `kraken` integration allows you to monitor exchange rates on [kraken.com](https://www.kraken.com/).
+For a list of tradable asset pairs check this [this kraken support article](https://support.kraken.com/hc/en-us/articles/201893658-Currency-pairs-available-for-trading-on-Kraken).
+
+### Extra configuration of the integration
+
+You can configure the update rate and tracked asset pairs options through the integration options flow by clicking the gear icon on the top right of the integration details page.
```
Comment (frenck) [inline source/_integrations/kraken.markdown]:
```suggestion
{% include integrations/config_flow.md %}

{% include integrations/option_flow.md %}
```

File: source/_integrations/kraken.markdown [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ha_config_flow: true
+ha_quality_scale: gold
+ha_codeowners:
+  - '@eifinger'
+ha_domain: kraken
+ha_platforms:
+  - sensor
+---
+
+The `kraken` integration allows you to monitor exchange rates on [kraken.com](https://www.kraken.com/).
```
Comment (frenck) [inline source/_integrations/kraken.markdown]:
```suggestion
The Kraken integration allows you to monitor exchange rates on [kraken.com](https://www.kraken.com/).
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/kraken.markdown (hunk 2 above)
Comment (eifinger) [reply source/_integrations/kraken.markdown]: Done =)

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**